manifests/manifest-chrome-testing.json

```diff
@@ -56,6 +56,8 @@
     ],
     "optional_permissions": [
         "clipboardWrite",
+        "favicon",
+        "chrome://favicon/",
```

Comment:
Is this an mv2 permission? I cannot find documentation for it.

Reply:
`chrome://favicon/` is mv2 permission, it's a common host permission. favicon is mv3 permission